main.go

```diff
@@ -96,6 +113,7 @@ func main() {
 		OutputQueue:               outputQueue,
 		SearchAPIURL:              cfg.SearchAPIURL,
 		HierarchyBuiltProducer:    hierarchyBuiltProducer,
+		OTServiceName:			   cfg.OTServiceName,
```

Comment:
Could the indentation be fixed here please so the "cfg" lines up with the "hierarchy..." above.